src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -17,16 +17,19 @@
  */
 public class AddCommand extends UndoableCommand {
 
+    public static final String[] COMMAND_WORDS = {"add", "a", "+"};
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
+    public static final String MESSAGE_USAGE = concatenateCommandWords(COMMAND_WORDS)
+            + ": Adds a person to the address book. "
             + "Parameters: "
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
             + "[" + PREFIX_TAG + "TAG]...\n"
-            + "Example: " + COMMAND_WORD + " "
+            + "Example: "
+            + COMMAND_WORD + " "
```

Comment:
why this change? to me this reduces readability

Reply:
i originally applied the concatenateCommandWords method to all COMMAND_WORD which led to the lines being extremely long and so i put them on a new line to increase readability, however i reverted on COMMAND_WORD that were not the first, but i did not put them back on the original line again, thus this. Ill include it in my next commit